### PR TITLE
sql model serializer tweaks

### PR DESCRIPTION
**Pull request by snopoke**
@emord 

cc @benrudolph

### CODE REVIEW

**Comment by emord** [comment]
Looks like build fails may be related. (All python builds failed)

File: corehq/form_processor/models.py [hunk 1]
```diff
@@ -263,6 +263,13 @@ def is_deleted(self):
         return self.state & self.DELETED == self.DELETED
 
     @property
+    def doc_type(self):
+        from corehq.form_processor.backends.sql.dbaccessors import doc_type_to_state
+        if self.is_deleted:
+            return 'XFormIntance' + DELETED_SUFFIX
```
Comment (benrudolph) [inline corehq/form_processor/models.py]: `XFormInstance`

File: corehq/form_processor/utils/xform.py [hunk 2]
```diff
@@ -161,18 +159,3 @@ def adjust_datetimes(data, parent=None, key=None):
     # return data, just for convenience in testing
     # this is the original input, modified, not a new data structure
     return data
-
-
-def add_couch_properties_to_sql_form_json(sql_form_json):
```
Comment (benrudolph) [inline corehq/form_processor/utils/xform.py]: nice

**Comment by benrudolph** [comment]
👍 on ✔️ the tests might be a bit rough to fix since i think in a lot of the tests we remove the doc_type from the expected output

File: corehq/form_processor/serializers.py [hunk 3]
```diff
@@ -48,20 +32,22 @@ class Meta:
...
     form = serializers.JSONField(source='form_data')
     auth_context = serializers.DictField()
     openrosa_headers = serializers.DictField()
-    attachments = serializers.JSONField(source='serialized_attachments')
+    external_blobs = serializers.JSONField(source='serialized_attachments')
 
     class Meta:
         model = XFormInstanceSQL
-        exclude = ('id',)
+        exclude = ('id','form_id')
```
Comment (dimagimon) [inline corehq/form_processor/serializers.py]: E231 missing whitespace after ','

**Comment by benrudolph** [comment]
```
======================================================================
ERROR: test_iteration (corehq.util.tests.test_doc_processor.XFormResumableSqlModelIteratorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 214, in test_iteration
    self.assertEqual([doc["_id"] for doc in self.itr], self.all_doc_ids)
  File "/mnt/commcare-hq-ro/corehq/util/doc_processor/sql.py", line 66, in __iter__
    yield reindex_accessor.doc_to_json(doc)
  File "/mnt/commcare-hq-ro/corehq/util/tests/test_doc_processor.py", line 173, in doc_to_json
    return self.wrapped_accessor.doc_to_json(doc)
  File "/mnt/commcare-hq-ro/corehq/form_processor/backends/sql/dbaccessors.py", line 118, in doc_to_json
    from corehq.form_processor.utils.xform import add_couch_properties_to_sql_form_json
ImportError: cannot import name add_couch_properties_to_sql_form_json
```

**Comment by snopoke** [comment]
:white_check_mark:

File: corehq/form_processor/models.py (hunk 1 above)
Comment (benrudolph) [inline corehq/form_processor/models.py]: this should probably be fixed before merge right? won't this mean we'll have incorrect doc types for some deleted forms in elastic?

Comment (snopoke) [inline corehq/form_processor/models.py]: c72043f thanks

**Pull request merged**